tests/foreman/ui_airgun/test_usergroup.py

```diff
@@ -0,0 +1,115 @@
+# -*- encoding: utf-8 -*-
+"""Test class for UserGroup UI
+
+:Requirement: Usergroup
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from fauxfactory import gen_string, gen_utf8
+from nailgun import entities
+
+from robottelo.datafactory import generate_strings_list
+from robottelo.decorators import (
+    parametrize,
+    tier1,
+    tier2,
+    upgrade,
+)
+
+
+@tier1
```

Comment:
remove tier1 decorator as we do not port or introduce tier1 tests. Let's keep all other your changes as it is, just remove decorators